IntegrationTests/ReducedConfig/IRtoKF/firmware/scripts/README.md

```diff
@@ -0,0 +1,59 @@
+These scripts are all triggered by the [Makefile](https://github.com/cms-L1TK/firmware-hls/blob/master/IntegrationTests/ReducedConfig/IRtoKF/firmware/Makefile). There should be no need to use these manually.
```

Comment:
a) Mention that the Makefiles compiles the code with IPBB, using the dependency files in IRtoKF/firmware/cfg/ .
b) Mention that IRtoKF/firmware/hdl/payload.vhd is the top-level interface to the EMP, and it calls the L1 tracking VHDL (SectorProcessor + KF).
c) Mention that firmware-hls/KalmanFilter is a git sub-module link to the repo containing the KF FW.

Reply:
I addressed b) and c); however, I frankly couldn't find in the makefile where EMP is called so I did not address a).

Reply:
Any L1Trk-EMP doc that is independent of specific chains should be placed in the EMP section of https://github.com/cms-L1TK/firmware-hls/blob/doc_emp/README.md . I've therefore moved the sentence about the KalmanFilter git submodule to this file. Has anyone managed to follow the instructions in this README? 

It's not obvious to me: (1) what " <EMP build path>" is (perhaps IntegrationTests/ReducedConfig/IRtoKF ?); (2) where input.txt should come from, (though I guess its by running https://github.com/cms-L1TK/firmware-hls/blob/doc_emp/IntegrationTests/ReducedConfig/IRtoKF/firmware/scripts/convert_emData2EMP_Link.py on our normal txt files); (3) where one specifies the chain to run (unless it's via "EMP build path"); (4) how to validate the output txt files. I guess one uses fwtosim_comparison.py , but https://github.com/cms-L1TK/firmware-hls/blob/doc_emp/IntegrationTests/ReducedConfig/IRtoKF/firmware/scripts/README.md suggests it is buggy.

It also looks to me as if https://github.com/cms-L1TK/firmware-hls/blob/doc_emp/IntegrationTests/ReducedConfig/IRtoKF/firmware/Makefile is compiling the L1Trk code without using EMP or IPBB. So I suspect the first two sentences in https://github.com/cms-L1TK/firmware-hls/blob/doc_emp/IntegrationTests/ReducedConfig/IRtoKF/firmware/scripts/README.md are wrong.

Reply:
Sorry guys, I dont know why I didnt receive any notification about these comments.

Here's the makefile I was referring to: https://github.com/cms-L1TK/firmware-hls/blob/doc_emp/IntegrationTests/ReducedConfig/IRtoKF/firmware/Makefile

Reply:
@FilMarini Good to hear from you. Am I correct that https://github.com/cms-L1TK/firmware-hls/blob/doc_emp/IntegrationTests/ReducedConfig/IRtoKF/firmware/Makefile makes no use of IPBB or EMP?

Reply:
@FilMarini In the file https://github.com/cms-L1TK/firmware-hls/blob/doc_emp/README.md , find the line containing "EMP build path". What value should we assign to "EMP build path"? Should we set "EMP build path" = "IntegrationTests/ReducedConfig/IRtoKF"?

Reply:
@tomalin 
Question 1: Thats right, it does not use IPBB or EMP per se, but it generates/modifies files in order to make them comply with IPBB/EMP

Question 2: The EMP build path depends on the type of configuration you want to build. For the skinny chain, the EMP build path to use is described here: https://github.com/cms-L1TK/firmware-hls/blob/doc_emp/README.md?plain=1#L376

Reply:
Ah, so we should change this line https://github.com/cms-L1TK/firmware-hls/blob/doc_emp/README.md?plain=1#L376 , so that it says "     *EMP build pat:h: `IntegrationTests/ReducedConfig/IRtoKF`"?

Reply:
@tomalin Yes I guess thats better

Reply:
Thanks. I've now made that change to README.